### PR TITLE
Added cylinder shape type

### PR DESCRIPTION
**Pull request by ceriottm**
Very simple, self-contained PR to include a cylinder shape - started off to switch to cylinder when headLength was zero, but it seems cleaner to have a separate object even if it was a bit more coding.

### CODE REVIEW

**Comment by ceriottm** [comment]
Wanted to make a path integral visualization and got a bit dragged away. Overall these are just a few improvements to the shapes functionality:
* I changed the way shapes are displayed so it is much faster to add many - basically all the vertices are added at once rather than one shape at a time
* I set things up so that when there are atom-based shapes with no explicit color definition the shapes will take the color of the corresponding atom.
* I added a utility to make path integral visualizations.
Ready to go, overall.

File: python/chemiscope/structures/_ase.py [hunk 1]
```diff
@@ -499,3 +499,84 @@ def _get_shape_params_atom(prefix, shape_kind, dictionary, atom_i):
         )
 
     return shape
+
+
+def ase_make_paths(bead_frames):
```
Comment (Luthaf) [inline python/chemiscope/structures/_ase.py:504]: Not a fan of this name, it's a bit too generic for my taste 😄. How about 

```suggestion
def ase_merge_pimd_frames(bead_frames):
```

Or something like it?

**Review by Luthaf — COMMENTED** [review]
Could you also add the new options to the input reference?

**Comment by ceriottm** [comment]
OK all done. @Luthaf  - since Rose has things to take care of these days, if you're happy I'd merge and move on.

File: python/chemiscope/structures/_ase.py [hunk 2]
```diff
@@ -499,3 +499,84 @@ def _get_shape_params_atom(prefix, shape_kind, dictionary, atom_i):
...
 
     return shape
+
+
+def ase_merge_pi_frames(bead_frames):
+    """
+    Takes a list of lists of ase.Atoms objects, corresponding to
+    the trajectories of the beads of a path integral calculation.
+    Returns a consolidated trajectory in which all beads are in the
+    same frame, together with propertie and shapes that help visualize
```
Comment (rosecers) [inline python/chemiscope/structures/_ase.py:509]:
```suggestion
    same frame, together with properties and shapes that help visualize
```

**Pull request merged**